src/controller/src/commands/synchronize_system_status_command.cpp

```diff
@@ -0,0 +1,24 @@
+#include <goliath/controller/commands/synchronize_system_status_command.h>
+
+#include <fstream>
+
+using namespace goliath::commands;
+using namespace goliath;
+
+const std::string SynchronizeSystemStatusCommand::TEMPERATURE_FILE = "/sys/devices/virtual/thermal/thermal_zone8/temp";
```

Comment:
Needs to be changed to `/sys/class/thermal/thermal_zone0/temp` for the Raspberry Pi.